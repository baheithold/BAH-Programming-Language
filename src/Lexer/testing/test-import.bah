# This file tests valid import statement


import "../this/is/a/file/path/filename.bah"
