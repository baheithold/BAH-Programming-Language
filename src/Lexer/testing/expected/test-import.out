IMPORT
STRING_TYPE "../this/is/a/file/path/filename.bah"
END_OF_INPUT
